packages/scheduler/src/components/FullCalendar/FullCalendar.js

```diff
@@ -68,16 +68,15 @@ class FullCalendar extends React.Component {
     },
     eventClick: event => this.props.onEventClick(event),
     eventRender: (event, element) => {
-      const content = <div>
-        <p>{event.description}</p>
+      const reservation = event.start.twix(event.end).format({monthFormat: 'MMMM', dayFormat: 'Do'})
+      const tooltipContent1 = <p dangerouslySetInnerHTML={{__html: `
+        <p>${event.description}</p>
         <p>
-          <i className="fa fa-clock-o" style={{paddingRight: '10px'}} aria-hidden="true"></i>
-          {event.start.twix(event.end).format({monthFormat: 'MMMM', dayFormat: 'Do'})}
-        </p>
-        <p>
-          <Conflict conflictStatus={event.conflict} intl={this.props.intl}/>
-        </p>
-      </div>
+          <i class="fa fa-clock-o" style="padding-right:10px" aria-hidden="true"></i>${reservation}
+        </p>`
+      }} />
+
+      const tooltipContent2 = <p><Conflict conflictStatus={event.conflict} intl={this.props.intl}/></p>
```

Comment:
are there better names for these variables than `tooltipContent1` and `tooltipContent2`? or maybe put all elements in an array and then build the content like this:

```
elements.map(element => ReactDOMServer.renderToString(element)).join('')
```